tests/core/pyspec/eth2spec/test/phase0/unittests/validator/test_validator_unittest.py

```diff
@@ -513,3 +513,12 @@ def test_compute_subscribed_subnets_random_2(spec):
 def test_compute_subscribed_subnets_random_3(spec):
     rng = random.Random(3333)
     run_compute_subscribed_subnets_arguments(spec, rng)
+
+
+@with_phases([PHASE0, ALTAIR, BELLATRIX, CAPELLA])
+@spec_state_test
+def test_slashed_validator_elected_for_proposal(spec, state):
+    proposer_index = spec.get_beacon_proposer_index(state)
+    state.validators[proposer_index].slashed = True
+
+    assert spec.get_beacon_proposer_index(state) == proposer_index
```

Comment:
This test case actually works for DENEB and EIP-6988 now because the default `state` is genesis and `state.latest_block_header.slot` is `0`.

We should skip one slot first.


```suggestion
@with_phases([PHASE0, ALTAIR, BELLATRIX, CAPELLA, DENEB])
@spec_state_test
def test_slashed_validator_elected_for_proposal(spec, state):
    next_slot(spec, state)
    proposer_index = spec.get_beacon_proposer_index(state)
    state.validators[proposer_index].slashed = True

    assert spec.get_beacon_proposer_index(state) == proposer_index
```